plugins/woocommerce/tests/e2e-pw/tests/merchant/create-cart-block.spec.js

```diff
@@ -0,0 +1,96 @@
+const { test, expect, request } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+const { closeWelcomeModal } = require( '../../utils/editor' );
+
+const transformedCartBlockTitle = 'Transformed Cart';
```

Comment:
Can you use an unique page name, to ensure the right page gets tested on test re-runs? 

```suggestion
const transformedCartBlockTitle = `Transformed Cart ${ Date.now() }`;
```

Reply:
Thanks for reviewing @adimoldovan , added as suggested. 👍